addons/website_event/controllers/main.py

```diff
@@ -309,7 +345,10 @@ def _process_attendees_form(self, event, form_details):
                 registration_index, field_name = key_values
                 if field_name not in registration_fields:
                     continue
-                registrations.setdefault(registration_index, dict())[field_name] = int(value) or False
+                reg = registrations.setdefault(registration_index, dict())
+                reg[field_name] = int(value) or False
+                if form_details.get('event_slot_id'):
+                    reg['event_slot_id'] = int(form_details['event_slot_id'])
```

Comment:
Why does it need special handling? Can't it be added inside "_get_website_registration_allowed_fields" ?

Should work the same way as "event_ticket_id" right?